core/src/main/java/hudson/model/AbstractProject.java

```diff
@@ -323,7 +323,12 @@ public void onLoad(ItemGroup<? extends Item> parent, String name) throws IOExcep
         builds = buildMixIn.getRunMap();
         triggers().setOwner(this);
         for (Trigger t : triggers()) {
-            t.start(this, Items.currentlyUpdatingByXml());
+            try {
+                t.start(this, Items.currentlyUpdatingByXml());
+            } catch (Exception e) {
+                Logger.getLogger(AbstractProject.class.getName()).log(Level.WARNING,
+                        "could not start trigger '" + t + "' while loading project '" + name + "'", e);
```

Comment:
@jglick As it uses the server's locale, it's in the preferred language of the admins, and not some user's locale nobody can read. And then class name and hex string of the object seem worse IMO.